modules/installation-configuration-parameters.adoc

```diff
@@ -729,44 +729,44 @@ ifdef::ibm-power-vs[]
 |platform:
   powervs:
     serviceInstanceGUID:
-|The ServiceInstanceGUID is the ID of the Power IAAS instance created from the {ibm-cloud-name} Catalog.
+|Specifies the ID of the Power IAAS instance created from the {ibm-cloud-name} Catalog.
 |String. For example, `existing_service_instance_GUID`.
 
 |platform:
   powervs:
     clusterOSImage:
-|The ClusterOSImage is a pre-created {ibm-power-server-name} boot image that overrides the default image for cluster nodes.
+|Specifies a pre-created {ibm-power-server-name} boot image that overrides the default image for cluster nodes.
```

Comment:
This is deprecated and is actually a noop at this point. Can we specify that in this PR or should it be separate?